### PR TITLE
updated model to take into account data sources

### PR DESCRIPTION
**Pull request by fvanderbiest**
This is a proposal for https://github.com/GFDRR/thinkhazard_processing/issues/14
Please review.

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Note that this will probably break https://github.com/GFDRR/thinkhazard/blob/master/thinkhazard/views/report.py and the tests (https://github.com/GFDRR/thinkhazard/blob/master/thinkhazard/tests/__init__.py)

**Comment by pgiraud** [comment]
Looks good to me. Any reason why you use `back_populates` instead of `backref`?
I agree that the thinkhazard report will be broken.

**Comment by fvanderbiest** [comment]
> Any reason why you use back_populates instead of backref?

I just followed the documentation, but you're right that backref allows for a shorter (maybe less explicit) syntax.

Cf "when the backref keyword is used on a single relationship, it’s exactly the same as if two relationships were created individually using back_populates on each" from http://docs.sqlalchemy.org/en/rel_1_0/orm/backref.html#linking-relationships-with-backref

File: thinkhazard_common/utils.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from models import HazardCategoryAdministrativeDivisionAssociation
+
+
+def associate_admindiv_hazardcategory(admindiv, hazardcategory, source):
```
Comment (arnaud-morvan) [inline thinkhazard_common/utils.py:4]: this is not necessary for sql version of the decision tree

**Comment by arnaud-morvan** [comment]
Please merge

**Pull request merged**